src/Workspaces/Core/Desktop/Workspace/FileTextLoader.cs

```diff
@@ -101,6 +101,34 @@ protected virtual SourceText CreateText(Stream stream, Workspace workspace)
             return textAndVersion;
         }
 
+        internal override TextAndVersion LoadTextAndVersionSynchronously(Workspace workspace, DocumentId documentId, CancellationToken cancellationToken)
```

Comment:
Why do we prefer the suffix Syncronously here instead of something htat indicates the retry nature like WithRetry?